managed/src/main/java/com/yugabyte/yw/commissioner/tasks/DestroyUniverse.java

```diff
@@ -45,6 +55,7 @@ public void run() {
     try {
       // Create the task list sequence.
       subTaskGroupQueue = new SubTaskGroupQueue(userTaskUUID);
+      tableManager = Play.current().injector().instanceOf(TableManager.class);
```

Comment:
Btw, DeleteBackup initializes tableManager in the `initialize` function. I don't know enough pros and cons of such approach. Maybe @Arnav15 can suggest. But anyway, if you leave it here you don't need for a class field.